portfolio/src/main/webapp/gallery.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta name="viewport" content="width=device-width, initial-scale=1">
```

Comment:
nit: be consistent with whether to use spaces or tabs so that content in your repository looks consistent. At Google typically we only use spaces